index.html

```diff
@@ -0,0 +1,198 @@
+    <div class="main-program-card-container">
+    <div class="main-program-card">
+      <div>
+        <i class="fa-solid fa-award main-program-image"></i> 
+      </div>
+      <div class="main-program-card-title">
+        <h4>Award</h4>
+      </div>
+      <div class="main-program-card-description">
+        <p>
+          Lorem ipsum dolor sit, amet consectetur
+          adipisicing elit. Nostrum minima.
+        </p>
+      </div>
+    </div>
+
+    <div class="main-program-card">
+        <div class="">
+            <i class="fa-solid fa-place-of-worship main-program-image"></i>
+        </div>
+        <div class="main-program-card-title">
+          <h4>Symposia</h4>
+        </div>
+        <div class="main-program-card-description">
+          <p>
+            Lorem ipsum dolor sit, amet consectetur
+            adipisicing elit. Nostrum minima.
+          </p>
+        </div>
+      </div>
+
+      
+      <div class="main-program-card">
+        <div class="">
+            <i class="fa-solid fa-people-arrows main-program-image"></i>
+        </div>
+        <div class="main-program-card-title">
+          <h4>Reachout</h4>
+        </div>
+        <div class="main-program-card-description">
+          <p>
+            Lorem ipsum dolor sit, amet consectetur
+            adipisicing elit. Nostrum minima.
+          </p>
+        </div>
+      </div>
+
+      
+      <div class="main-program-card">
+        <div class="">
+            <i class="fa-solid fa-plane-arrival main-program-image"></i>
+        </div>
+        <div class="main-program-card-title">
+          <h4>Home-coming</h4>
+        </div>
+        <div class="main-program-card-description">
+          <p>
+            Lorem ipsum dolor sit, amet consectetur
+            adipisicing elit. Nostrum minima.
+          </p>
+        </div>
+      </div>
+
+      
+      <div class="main-program-card">
+        <div class="">
+            <i class="fa-solid fa-business-time main-program-image"></i>
+        </div>
+        <div class="main-program-card-title">
+          <h4>Business talk</h4>
+        </div>
+        <div class="main-program-card-description">
+          <p>
+            Lorem ipsum dolor sit, amet consectetur
+            adipisicing elit. Nostrum minima.
+          </p>
+        </div>
+      </div>
+    </div>
+    <p class="see-whole-program"><a href="#"> SEE THE WHOLE PROGRAM</a></p>
+    <div class="main-program-card-btn"><button class="join-btn" type="button">Join GSSO Alumni Club</button></div>
+  </section>
+  <section class="featured-speakers">
+    <div class="featured-speakers-title">
+      <h1>Featured Speakers</h1>
+      <hr class="main-hr">
+    </div>
+    
+<div id="featured-speakers-card-container">
+
+</div>
+    <div class="featured-speakers-card-button">
+        <button id="more" type="button" onclick="viewMore()">MORE <i class="fa-solid fa-angle-down"></i></button>
+    </div>
+  </section>
+
+<section class="desktop-footer">
```

Comment:
Kindly replace this `section` tag with `footer` tag since this block is the footer of your page 🤗